pypeit/core/findobj_skymask.py

```diff
@@ -1325,7 +1338,7 @@ def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, slit_spat_id, order
 
     return sobjs_ech
 
-
+# TODO: THIS IS DEPRECATED
```

Comment:
Can we comment it out, or move it to a deprecated file, or simply delete it?

Reply:
Moved to deprecated.